src/main/java/org/folio/consortia/messaging/listener/ConsortiaEventListener.java

```diff
@@ -1,17 +1,19 @@
 package org.folio.consortia.messaging.listener;
 
-import static org.folio.consortia.utils.TenantContextUtils.getFolioExecutionContextCreatePrimaryAffiliationEvent;
-import static org.folio.consortia.utils.TenantContextUtils.getFolioExecutionContextDeletePrimaryAffiliationEvent;
-import static org.folio.consortia.utils.TenantContextUtils.runInFolioContext;
-
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.consortia.service.ConfigurationService;
 import org.folio.consortia.service.UserAffiliationService;
 import org.folio.spring.FolioModuleMetadata;
+import org.folio.spring.integration.XOkapiHeaders;
+import org.folio.spring.scope.FolioExecutionContextSetter;
 import org.springframework.kafka.annotation.KafkaListener;
 import org.springframework.messaging.MessageHeaders;
 import org.springframework.stereotype.Component;
 
-import lombok.RequiredArgsConstructor;
-import lombok.extern.log4j.Log4j2;
+import static org.folio.consortia.utils.TenantContextUtils.createFolioExecutionContext;
+import static org.folio.consortia.utils.TenantContextUtils.getHeaderValue;
+import static org.folio.consortia.utils.TenantContextUtils.runInFolioContext;
```

Comment:
please check your ide settings. Import static should be on the top

Reply:
I didn't configure these import static and I think it cannot be configured it should be top or below. maybe they should change their policy based on mostly used imports style. I am using 2022.3 version. Please let me know if yours is also this version, I will check again.

Reply:
please try installing formatter plugin according to following wiki page https://wiki.folio.org/display/FOLIJET/Java+code+formatter+for+IDEs